packages/lodestar-cli/src/cmds/account/cmds/wallet/create.ts

```diff
@@ -59,14 +59,21 @@ export async function handler(options: IWalletCreateOptions): Promise<void> {
   // Create a new random mnemonic.
   const mnemonic = bip39.generateMnemonic();
 
-  // Create a random password if the file does not exist.
+  if (path.parse(passphraseFile).ext !== ".pass") {
+    throw new YargsError("passphraseFile must end with .pass, make sure to not provide the actual password");
+  }
   if (!fs.existsSync(passphraseFile)) {
-    if (path.parse(passphraseFile).ext !== ".pass") {
-      throw new YargsError("passphraseFile must end with .pass, make sure to not provide the actual password");
-    }
     writeFile600Perm(passphraseFile, randomPassword());
   }
-  const password = fs.readFileSync(passphraseFile, "utf8");
+  const password = readPassphraseFile(passphraseFile);
+  // Validate the passphraseFile contents to prevent the user to create a wallet with a password
+  // that is the contents a random unintended file
+  try {
+    if (password.includes("\n")) throw Error("contains multiple lines");
+    if (password.length > 512) throw Error("is really long");
```

Comment:
where do we get the 512 number from?  also, could we perhaps move it to a constants file?

Reply:
It's a completely arbitrary number, it just seemed like something long enough that would never be used as a password. Happy to change it to something else.

Reply:
@3xtr4t3rr3str14l I've added a comment explaining the arbitrary nature of 512.